pkgs/test/README.md

```diff
@@ -221,6 +221,22 @@ The files can then be formatted using the `package:coverage`
 Coverage gathering is currently only implemented for tests run on the Dart VM or
 Chrome.
 
+Here's an example of how to run tests and format the collected coverage to LCOV:
+
+```shell
+## Run Dart tests and output them at directory `./coverage`:
+dart run test --coverage=./coverage
+
+## Format collected coverage to LCOV (only for directory "lib")
+pub global run coverage:format_coverage --packages=.packages --report-on=lib --lcov -o ./coverage/lcov.info -i ./coverage
+
+## Generate LCOV report:
+genhtml -o ./coverage/report ./coverage/lcov.info
```

Comment:
I'm not sure we reference this tool anywhere else? It looks like it's a linux only thing? Should we clarify here?

Reply:
Maybe add a link to LCOV project:

https://github.com/linux-test-project/lcov

Reply:
Sure. Just make it clear that this is one such optional tool.